### DISCUSSION TITLE
Automate creation of sub-account within a directory?

### DISCUSSION THREAD

**Discussion by dachtera** [SAP-archive/btp-setup-automator#270] [Q&A] [answered]
I have not seen how to create a sub-account within a provided directory?  We need to create sub-accounts within Customer directories (to help organize our efforts) and would prefer to not manually have to move sub-accounts once created with the automator.
![Screen Shot 2022-10-19 at 10 41 28 AM](https://user-images.githubusercontent.com/115640072/196739346-90473bac-8e5d-47bb-8850-5428a318ec03.png)

Preferred expectation using automator:

![Screen Shot 2022-10-19 at 10 42 08 AM](https://user-images.githubusercontent.com/115640072/196739528-8590b819-2fcf-4b6a-9b68-5f27645d2b65.png)

**Comment by lechnerc77** [comment]
The creation of directories is not supported including the assignment to an existing directory. Please open a [feature request](https://github.com/SAP-samples/btp-setup-automator/issues/new?assignees=&labels=enhancement&template=feature-request.yml&title=%5BFEATURE+REQUEST%5D+%3Ctitle%3E). Also happy to receive a PR for this enhancement

**Comment by lechnerc77** [comment, marked as answer]
Feature Request is open: <https://github.com/SAP-samples/btp-setup-automator/issues/272>

**Reply by lechnerc77** [reply]
Temporal workaround is described in the issue.

**Reply by lechnerc77** [reply]
Enhanced with PR #272
